### ISSUE TITLE
WCAG 2.1 Understanding has WCAG 2.2 language

### ISSUE THREAD

**Issue by yatil** [w3c/wcag#1162]
This page: https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html

has the following language on it:

> Note that a keyboard focus indicator can take different forms. This criterion does not specify what the form is, but [Focus Visible (Enhanced)](#) does define how visible the indicator should be. Passing [Focus Visible (Enhanced)](#) would pass this success criterion.

Both links to “Focus Visible (Enhanced)” link to an empty fragment identifier (#). 

This is extremely confusing for people who follow WCAG 2.1. The Understanding pages should clearly indicate what is applicable for which version of WCAG.

**Comment by patrickhlauke** [comment]
This is another symptom of the current mashing up of the same pages reused across different versions. Particularly now that 2.x versions are drifting/changing across WCAG versions, it's probably time to consider setting up multiple completely independent versions of understanding/techniques/etc. This would avoid these sorts of version bleeds...

**Comment by patrickhlauke** [comment]
also note that that page has "Success Criterion 2.4.7 Focus Visible (Level A)" ... but under 2.1, it's Level AA. again, problem caused by trying to reuse things...

**Comment by yatil** [comment]
Thanks for seeing the level change, @patrickhlauke!

For years I’m now redirecting people to the WCAG 2.1 Understanding and Techniques as they often work off some 2.0 Understanding documents from 2008, as they are still bookmarked or easily findable through Google but not updated anymore by the Working Group. When SCs are the same in 2.0 and 2.1 that is not a problem. But I would like for the WG to put in proper redirects.

I suppose the change from Level AA to Level A also means that some of the sufficient techniques might be not sufficient anymore or that some of the best practices (“advisory”) techniques might become sufficient techniques.

All that needs to be clearly communicated to the readers of the documents. 

I get almost daily complaints over the usefulness of those resources. Much of the wording is indecipherable without deep WCAG knowledge and the examples are often outdated and not helpful.

**Comment by alastc** [comment]
It is tricky, as it stands we have 1 version of the understanding/techniques docs (apart from the old TR docs), so anything updated is then shown. (You'll see a span with class of "wcag22" which isn't actually doing anything because there is only one version...)

However, looking ahead, will it be more confusing to have different versions of the same documents?

Or is it better to have one version with certain bits highlighted as differences?

**Comment by yatil** [comment]
I don’t care about the exact mechanism, it is just important that

1. Content that is specific to a certain version of WCAG is either available separately or highlighted appropriately.
2. The information on the pages is factually correct.

If the Understanding documents need to be one thing, it is to be super reliable. Wrong information easily spreads, especially when it is on normative-looking pages. (I’m aware that this should change hopefully soon.) Changes need to be explained carefully and all info must be especially clear. 

There are many examples of other documentation online that has versioning built-in sometimes with, sometimes without, separate pages for different versions: [PHP](https://www.php.net/manual/en/function.strftime), [eleventy](https://www.11ty.dev/docs/data-eleventy-supplied/), [Laravel](https://laravel.com/docs/7.x/responses), [Yii](https://www.yiiframework.com/doc/guide/2.0/en/structure-views)…

**Comment by bruce-usab** [comment]
@alastc, I would say a single version with highlights.  That would (1) lessen the problem of people finding out-of-date resources, and (2) maintenance should be a little simpler.

@yatil wrote:
> I suppose the change from Level AA to Level A also means that some of the sufficient techniques might be not sufficient anymore or that some of the best practices (“advisory”) techniques might become sufficient techniques.

I can't think of why / how this would be the case.  Do you have a theoretical example in mind?  (AAA SC are a little trickier, since  they are not required to be as widely applicable as A/AA SC.  But for a AAA SC to advance to AA, it would have to be one that is fully technology neutral.)

**Comment by patrickhlauke** [comment]
> However, looking ahead, will it be more confusing to have different versions of the same documents?

But that's the thing...they won't be exactly the same. And this could be at a fundamental level (e.g. the fact that 2.4.7 is AA in 2.0 and 2.1, but A in 2.2) that can't easily be handwaved/noted in a little block of text.

> Or is it better to have one version with certain bits highlighted as differences?

**Comment by yatil** [comment]
The wrong information is still posted online. Can we revert to a version that does not contain confusing (that sentence that links to nowhere) or wrong (WCAG level) information?

**Comment by yatil** [comment]
What’s the time line for this? I would like to point people to factually correct information again.

**Comment by michael-n-cooper** [comment]
The original issue filed was about a broken cross reference, which is fixed by the above commit. The issue forked into discussion of how to manage versioned Understanding files. That is a separate issue under discussion now, and not handled as part of this issue.

**Comment by yatil** [comment]
The original issue was also about generally having WCAG 2.2 verbiage on WCAG 2.1 Understanding documents without any marking.

**Comment by yatil** [comment]
I opened bugs for the other two – much more important, IMHO, issues: #1177 & #1178
